dapr/proto/runtime/v1/dapr.proto

```diff
@@ -687,6 +708,174 @@ message UnlockResponse {
   Status status = 1;
 }
 
+// SubtleGetKeyAlpha1Request is the request object for SubtleGetKeyAlpha1.
+message SubtleGetKeyAlpha1Request {
+  enum KeyFormat {
+    // PEM (PKIX) (default)
+    PEM = 0;
+    // JSON (JSON Web Key) as string
+    JSON = 1;
+  }
+ 
+  // Name of the component
+  string component_name = 1 [json_name="component"];
```

Comment:
```suggestion
  string component_name = 1 [json_name="componentName"];
```
Lets have the same name across both proto and json
same for other scenarios.

Reply:
Changed

(Although the JSON name here is unused)